samples/tetris/README.md

```diff
@@ -4,27 +4,20 @@ This example shows implementation of simple Tetris game using SDL
 (Simple DirectMedia Layer) library for rendering. SDL allows easy development
 of cross-platform game and multimedia applications.
 
-Start with building compiler by using `dist` and `cross_dist` for cross-targets (unless
-using binary distribution).
-
 Install SDL2 development files (see https://www.libsdl.org/download-2.0.php). For Mac -
 copy `SDL2.framework` to `$HOME/Library/Frameworks`. For Debian-like Linux -
 use `apt-get install libsdl2-dev`.
 For Windows - `pacman -S mingw-w64-x86_64-SDL2` in MinGW64 console, if you do
 not have MSYS2-MinGW64 installed - install it first as described in http://www.msys2.org
 
-To build Tetris application for your host platform (Mac or Linux) use `../gradlew assemble`. 
-    
-Aleternatively for Mac and Linux `./build.sh`.
+To build Tetris application for your host platform use `../gradlew assemble`.
 
-For Windows use `build.bat`.
-    
 Note that SDL2 must be installed on the host.
 
-Now you can run the game
+Now you can run the game using `../gradlew runProgram` or directly with
+
+     ./build/bin/tetris/main/release/executable/tetris.kexe
 
```

Comment:
Maybe write about Mojave bug smth like
On macOS 10.14 Mojave SDL window may render black until window is dragged, due to an SDL issue.

Reply:
Why do we need to mention about this if fix https://github.com/JetBrains/kotlin-native/commit/03f13833326ae60fa4ddf17899e6dbb4ac4e77fa is already in the source code?

Reply:
It still sometimes happens, I afraid.

Reply:
Done.